packages/Tabs/index.js

```diff
@@ -70,7 +85,8 @@ export const TabList = React.forwardRef(({ as, children, ...props }, ref) => {
 TabList.displayName = 'TabList'
 TabList.propTypes /* remove-proptypes */ = {
   as: oneOfType(COMPONENT_TYPE),
-  children: node
+  children: node,
+  orientation: oneOf(['veritcal', 'horizontal'])
```

Comment:
```suggestion
  orientation: oneOf(['vertical', 'horizontal'])
```

Reply:
Name: Code Spell Checker
Id: streetsidesoftware.code-spell-checker
Description: Spelling checker for source code
Version: 1.9.0
Publisher: Street Side Software
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

😛